nilearn/plotting/displays.py

```diff
@@ -775,6 +776,7 @@ def _show_colorbar(self, cmap, norm, threshold=None):
             self._colorbar_ax, ticks=ticks, norm=norm,
             orientation='vertical', cmap=our_cmap, boundaries=bounds,
             spacing='proportional', format='%.2g')
+        self._cbar.patch.set_facecolor(mid_gray)
```

Comment:
Is it this enough ? Do you also need adding (0.) ?

For me it looks like using only this ```self._cbar.patch.set_facecolor(mid_gray)``` is sufficient. Correct me if I am wrong.

Reply:
This sets the background color, if you add `(0.,)` it makes it transparent and thus defeats the purpose